Appsilon/rhino
# Box import separate packages modules

### Changes
- Closes #541 
- Adds a `box_separate_calls_linter()`
- Adds the litner to the `.lintr` template
- Adds documentation

### How to test
1. Unit tests for `box_separate_calls_linter()` exist.
2. Locally install `rhino`, version should be `1.6.0.9004`, initialize a new rhino application then proceed to declare box imports.
3. Add a `box::use()` with both package and modules.
4. `rhino::lint_r()` should find and point to the invalid `box::use()` call.
